dot/sync/chain_sync.go

```diff
@@ -525,6 +462,81 @@ func (cs *chainSync) maybeSwitchMode() {
 	}
 }
 
+func (cs *chainSync) handleResult(res *worker) error {
```

Comment:
nit rename `res` to something more explicit? Maybe `resultWorker`?

Reply:
Updated.